packages/web3-eth-personal/test/integration/personal.test.ts

```diff
@@ -0,0 +1,125 @@
+/*
+This file is part of web3.js.
+
+web3.js is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+web3.js is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with web3.js.  If not, see <http://www.gnu.org/licenses/>.
+*/
+import { isHexStrict } from 'web3-validator';
+import { toChecksumAddress } from 'web3-utils';
+import { EthPersonal } from '../../src/index';
+import { importedAccount } from '../config/personal.test.config';
+import {
+	getSystemTestBackend,
+	getSystemTestAccounts,
+	getSystemTestProvider,
+	itIf,
+} from '../fixtures/system_test_utils';
+
+describe('personal integration tests', () => {
+	let ethPersonal: EthPersonal;
+	let accounts: string[];
+	beforeAll(() => {
+		const clientUrl = getSystemTestProvider();
+		ethPersonal = new EthPersonal(clientUrl);
+	});
+	beforeEach(async () => {
+		accounts = await getSystemTestAccounts();
+	});
+	it('new account', async () => {
+		const newAccount = await ethPersonal.newAccount('!@superpassword');
+		expect(isHexStrict(newAccount)).toBe(true);
```

Comment:
```suggestion
		expect(isHexStrict(newAccount)).toBeTruthy();
```
The linter should've complained about this :thinking: